Sources/MainUnit.pas

```diff
@@ -203,6 +208,77 @@ procedure TMainForm.AboutActionExecute(Sender: TObject);
   AboutBox.ShowModal;
 end;
 
+procedure TMainForm.actLoadPGNFileExecute(Sender: TObject);
```

Comment:
1. It is not a good idea to mix GUI creation and parsing in one function. Parsing should go to some other unit (like `NotationLists.pas`, `ChessNotation.pas` and the related things are done). GUI creation should go to the new form unit itself instead of creating everything in the main form.
2. I don't think that splitting PGNs with regular expressions is reliable. While Chess256 parser currently assumes that there is only one PGN in the file, the splitter ideally should use this parser, without relying on regular expressions.